tockloader-lib/src/tabs/tab.rs

```diff
@@ -2,13 +2,14 @@
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 // Copyright OXIDOS AUTOMOTIVE 2024.
 
+use crate::board_settings::BoardSettings;
 use crate::errors::{TabError, TockloaderError};
 use crate::tabs::metadata::Metadata;
 use std::fs::File;
 use std::io::Read;
 use tar::Archive;
 
-struct TbfFile {
+pub struct TbfFile {
     pub filename: String,
```

Comment:
```suggestion
    pub filename: TbfFilename,
```

This does mean we do this parsing in `Tab::open`